docs/src/conf.py

```diff
@@ -1,57 +1,80 @@
-#
-# For the full list of built-in configuration values, see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
-
 import os
-import re
+import sys
+from datetime import datetime
+
+import tomli  # Replace by tomllib from std library once docs are build with Python 3.11
+
+import metatensor_models
 
 
+ROOT = os.path.abspath(os.path.join("..", ".."))
+sys.path.insert(0, ROOT)
+
 # -- Project information -----------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'metatensor-models'
-copyright = '2023, metatensor-models developers'
-author = 'metatensor-models developers'
+# The master toctree document.
+master_doc = "index"
 
-# Parse the version from the module.
-with open(os.path.join(os.path.dirname(__file__), '..', '..', 'src', 'metatensor_models', '__init__.py')) as f:
-    version = re.match(r".*__version__ = \"(.*?)\"", f.read(), re.S).group(1)
-release = version
+with open(os.path.join(ROOT, "pyproject.toml"), "rb") as fp:
+    project_dict = tomli.load(fp)["project"]
 
+project = project_dict["name"]
+author = ", ".join(a["name"] for a in project_dict["authors"])
 
-# -- General configuration ---------------------------------------------------
+copyright = f"{datetime.now().date().year}, {author}"
 
-needs_sphinx = "4.0.0"
+# The full version, including alpha/beta/rc tags
+release = metatensor_models.__version__
 
-python_use_unqualified_type_names = True
+
+# -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx_rtd_dark_mode',
     "sphinx.ext.viewcode",
     "sphinx.ext.autodoc",
     "sphinx.ext.intersphinx",
     "sphinx_toggleprompt",
 ]
 
-default_dark_mode = True
+python_use_unqualified_type_names = True
 
 autoclass_content = "both"
 autodoc_member_order = "bysource"
 autodoc_typehints = "both"
 autodoc_typehints_format = "short"
 
 intersphinx_mapping = {
+    "ase": ("https://wiki.fysik.dtu.dk/ase/", None),
     "python": ("https://docs.python.org/3", None),
     "torch": ("https://pytorch.org/docs/stable/", None),
+    "metatensor": ("https://lab-cosmo.github.io/metatensor/latest/", None),
+    "rascaline": ("https://luthaf.fr/rascaline/latest/", None),
 }
 
-
 # -- Options for HTML output -------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = 'alabaster'
-html_static_path = ['_static']
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "furo"
+
+html_theme_options = {
+    "footer_icons": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/ceriottm/MeshLODE",
```

Comment:
No comment

Reply:
Sorry I forgot to add the toml parsing here.